README.md

```diff
@@ -167,8 +169,8 @@ Add CSS rules in your stylesheet.
 
 <h2 id="discord-community">Discord community</h2>
 <p>
-We have a Discord community for Devicons. You can easily request icons, discuss, and have a good time talking with the community members! Join today: https://discord.gg/hScy8KWACQ.
-<b>Note that the Discord server is unofficial, and Devicons is still being maintained via GitHub.</b>
+We have a Discord community for Devicons. You can easily request icons, discuss, and have a good time talking with the community members! Join today: https://discord.gg/hScy8KWACQ.</p>
+<p><b>Note that the Discord server is unofficial, and Devicons is still being maintained via GitHub.</b>
```

Comment:
Relevant, but for another discussion/issue: is the Discord server still unofficial? I think we should state it as official now.

Reply:
Whoops, didn't see this comment before merging, but yes... I agree. The Discord server should be considered official

Reply:
#1641 PR fixes this🙂, mentioning the server as official.